### PR TITLE
Runtime Options 

### PR DESCRIPTION
**Pull request by vsingh18567**
importing from [django-runtime-options](https://github.com/pennlabs/django-runtime-options)

### CODE REVIEW

File: pennlabs/options/models.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        if self.value_type == self.TYPE_BOOL:
+            return self.as_bool()
+        return self.value
+
+    @typed_value.setter
+    def typed_value(self, value):
+        """
+        Given a typed value, convert into string form and set the untyped variable.
+        """
+        self.value = str(value)
```
Comment (vsingh18567) [inline pennlabs/options/models.py:81]: the issue was that this function was previously not called `typed_value`, but to be a property.setter, the function needs to have the same name as the property.

File: .coveragerc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[run]
+omit=*apps.py
```
Comment (vsingh18567) [inline .coveragerc]: let me know if there is a better way of doing this than creating a `.coveragerc`

File: pennlabs/options/models.py (hunk 1 above)
Comment (ArmaanT) [reply pennlabs/options/models.py:81]: I feel so validated right now

File: .coveragerc (hunk 2 above)
Comment (ArmaanT) [reply .coveragerc]: There's a coverage section within `tox.ini`. You should be able to add the config you want there. We do something similar with `setup.cfg` in platform ([link](https://github.com/pennlabs/platform/blob/05e035c8ab0703aa34971ffed1819641b9783721/backend/setup.cfg#L15))

**Pull request merged**